drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -29,11 +45,12 @@ static void ite_intc_isr_clear(unsigned int irq)
 	if (irq > CONFIG_NUM_IRQS) {
 		return;
 	}
-	g = irq / 8;
-	i = irq % 8;
+	g = irq / MAX_REGISR_IRQ_NUM;
```

Comment:
Shouldn't this commit be squashed into the previous one? I am reviewing the same code twice?

Reply:
I am sorry to have confused you.
It seems like one of the commit cover other commit files. 
I will recommit them to fix this.